### PR TITLE
docs: light client extension helpers

### PR DESCRIPTION
**Pull request by ryanleecode**

### CODE REVIEW

File: packages/light-client-extension-helpers/README.md [hunk 1]
```diff
@@ -1 +1,210 @@
...
+
+  return { provider }
+}
+```
+
+### extensionPagesHelper
+
+The extension page helper is conceptually the same as the web page helper,
+except it is used in the user interface of your extension. By importing `helper`,
+you can access light client functionality
```
Comment (marshacb) [inline packages/light-client-extension-helpers/README.md:115]:
```suggestion
you can access light client functionality.
```

**Review by marshacb — APPROVED** [review]
question and small nit. lgtm

File: packages/light-client-extension-helpers/README.md [hunk 2]
```diff
@@ -1 +1,210 @@
 # @substrate/light-client-extension-helpers
+
+The `@polkadot-api/light-client-extension-helpers` library is designed to encapsulate complex problems solved over years of development, making it easier to build robust browser extensions. The library exposes four primary components:
```
[the remark this answers is not in the document]
Comment (ryanleecode) [reply packages/light-client-extension-helpers/README.md:3]: Sure

[the remark this answers is not in the document]
Comment (ryanleecode) [reply packages/light-client-extension-helpers/README.md:3]: Actually this section shall be deleted

**Pull request merged**